doc/user/content/guides/temporal-filters.md

```diff
@@ -3,8 +3,8 @@ title: "Temporal Filters"
 description: "Perform time-windowed computation over temporal data."
 weight:
 menu:
-  main:
-    parent: guides
+    main:
```

Comment:
Seeing a couple of your recent PRs and all the changes in indentation and string quoting, I think something's up with your editor settings!

Reply:
Yep. It seems that `Prettier` was refactoring them (no-side effects but I will dissable to make it easier to read the PR)